lang/attribute/account/src/lib.rs

```diff
@@ -289,7 +289,8 @@ pub fn zero_copy(
 
     proc_macro::TokenStream::from(quote! {
         #[derive(anchor_lang::__private::ZeroCopyAccessor, Copy, Clone)]
-        #[repr(C)]
+        #[cfg_attr(not(feature = "anchor-zero-copy-repr-packed"), repr(C))]
```

Comment:
Can we pass this in as an argument instead?

Reply:
discussed elsewhere: instead of argument, would be even better to give the user the choice to add a `repr`
if they add one, do nothing.
If they don't. Add `repr(c)`